src/test/scala/org/broadinstitute/dsde/firecloud/webservice/ShareLogApiServiceSpec.scala

```diff
@@ -28,21 +28,7 @@ final class ShareLogApiServiceSpec extends BaseServiceSpec with ShareLogApiServi
 
   override def beforeAll(): Unit = {
     mockSamServer = startClientAndServer(MockUtils.samServerPort)
-
-    // enabled user
-    mockSamServer
-      .when(request
-        .withMethod("GET")
-        .withPath("/register/user/v2/self/info"))
-      .respond(
-        org.mockserver.model.HttpResponse.response()
-          .withHeaders(MockUtils.header).withBody("""{
-                                                    |  "adminEnabled": true,
-                                                    |  "enabled": true,
-                                                    |  "userEmail": "enabled@nowhere.com",
-                                                    |  "userSubjectId": "enabled-id"
-                                                    |}""".stripMargin).withStatusCode(OK.intValue)
-      )
+    returnEnabledUser(mockSamServer)
```

Comment:
Were there changes to be made to the ShareLogApi as well, or just the Spec?  I see ShareLog was listed as an API that might need to be authenticated, but wasn't sure if it does

Reply:
done in #980

Reply:
Gotcha